apps/condo/domains/common/components/PostMessageProvider/validators.ts

```diff
@@ -84,4 +93,5 @@ export const validators: ValidatorsType = {
     CondoWebAppShowNotification: ajv.compile(CondoWebAppShowNotificationParamsSchema),
     CondoWebAppShowProgressBar: ajv.compile(CondoWebAppShowProgressBarParamsSchema),
     CondoWebAppUpdateProgressBar: ajv.compile(CondoWebAppUpdateProgressBarParamsSchema),
+    CondoWebAppCall: ajv.compile(CondoWebAppCallParamsSchema),
```

Comment:
Put it closer to analytics please...Since events are grouped by source + ordered alphabetically for better reading